pkg/controller/admissionchecks/multikueue/multikueuecluster.go

```diff
@@ -671,7 +671,6 @@ func (c *clustersReconciler) getRemoteClients() []*remoteClient {
 // +kubebuilder:rbac:groups="",resources=events,verbs=create;watch;update
 // +kubebuilder:rbac:groups=kueue.x-k8s.io,resources=multikueueclusters,verbs=get;list;watch
 // +kubebuilder:rbac:groups=kueue.x-k8s.io,resources=multikueueclusters/status,verbs=get;update;patch
-// +kubebuilder:rbac:groups=multicluster.x-k8s.io,resources=clusterprofiles,verbs=get;list;watch
```

Comment:
Should we also track only kueue-system in cache? 

Please test e2e if creating ClusterProfile instances works, I think currently it might be failing.

Similarly as we do here: https://github.com/kubernetes-sigs/kueue/pull/7188/files#diff-a3d824da3c42420cd5cbb0a4a2c0e7b5bfddd819652788a0596d195dc6e31fa5R106-R123

Reply:
Let me open an issue to add integration tests.